extensions/amp-subscriptions/0.1/subscription-platform.js

```diff
@@ -41,6 +41,18 @@ export class SubscriptionPlatform {
    * @param {!./amp-subscriptions.RenderState} unusedRenderState
    */
   activate(unusedRenderState) {}
+
+  /**
+   * Returns if pingback is enabled for this platform
+   * @returns {boolean}
+   */
+  isPingbackEnabled() {}
+
+  /**
+   * Performs the pingback to the subscription platform
+   * @param {!./entitlement.Entitlement} unusedSelectedPlatform
```

Comment:
It looks like it actually returns `!Promise|undefined`.

Reply:
No, `{Promise}` is not allowed at all, but indicates `{?Promise}` or nullable. `undefined` a bit different.